### PR TITLE
Novice Thaumaturgy Questbook Changes

### PR DESCRIPTION
**Pull request by PlayfulPiano**
- Increased "Your First Wand" icon size
- Modified required progression path to better separate required quests from secondary quests.
- Added a new quest for interacting with a Pech ("A Pechuliar Merchant"), can be used to unlock "Getting Shards".
- Added two new quest triggers for obtaining a Profane Wand ("A Dangerous Trade") and obtaining Scribing Tools early (requires Profane Wand trigger), unlocking the questpage if discovered.
- Separated "Useful Plant Life" quest into two quests, "A New Way to "Farm" Resources" & "Growing your own Knowledge!", with better description explanations for each.
- Swapped the farm related prerequisite of "Primal Shrooms" with "Enchanted Earth"
- Other description and/or quest progression tweaks

image
image
image
image
image
image

Skip Showcase:
image
image

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/NoQuestLine/TriggerPlatlineO-AAAAAAAAAAAAAAAAAAALXA==.json
Comment (FourIsTheNumber) [inline config/betterquesting/DefaultQuests/Quests/NoQuestLine/TriggerPlatlineO-AAAAAAAAAAAAAAAAAAALXA==.json:1]: This deletion looks unintended

Comment (PlayfulPiano) [reply config/betterquesting/DefaultQuests/Quests/NoQuestLine/TriggerPlatlineO-AAAAAAAAAAAAAAAAAAALXA==.json:1]: huh, you're correct. I didn't even touch this quest, weird. What would be the best way to fix this, on the PR side of things?

Comment (FourIsTheNumber) [reply config/betterquesting/DefaultQuests/Quests/NoQuestLine/TriggerPlatlineO-AAAAAAAAAAAAAAAAAAALXA==.json:1]: > huh, you're correct. I didn't even touch this quest, weird. What would be the best way to fix this, on the PR side of things?

Go to a fresh copy of the bq configs, go to that file path, copy the deleted quest back into your repo and then commit/push again

Comment (PlayfulPiano) [reply config/betterquesting/DefaultQuests/Quests/NoQuestLine/TriggerPlatlineO-AAAAAAAAAAAAAAAAAAALXA==.json:1]: done

**Review by chochem — CHANGES_REQUESTED** [review]
- Dont change the start. these are critical main quests. The requirements are intentional.
- Dont make a secret quest. People already hate the ones we have as they dont know how to finish the respective quest tab. Either this deserves a proper quest or not. Probably best to just use the trigger without that one.
- That processing infused stones quest looks totally useless. whats the point?
- Pech looks good. but we dont use meeting tasks anywhere else. have you tested that it works well?

**Comment by Dream-Master** [comment]
meeting quests are a nice idea if they working. never tested it.

**Comment by PlayfulPiano** [comment]
I'll have to test back when I get home from work today (assumed they already had precedent in the quest book, didn't realize this wasn't the case).

As for early progression changes, would it be better if the thaumometer quest was a checkbox/optional retrieval task combination? Main thing is so players who get an early wand before laser engraver can still progress in the quest book based on what they actually have access to (so workbench and crucible, where neither needs the thaumometer or shards).

As for the processing infused stones quest, that's meant to be an intermediate quest to store the old getting shards description for the typical LV progression path, while also allowing pechs to act as an alt progression path for pre LV shards.

**Comment by PlayfulPiano** [comment]
Also when I get back home might look into adding some more information about the strength/advantages of early thaumcraft progression (redstone, lapis, sugar sources) and possibly a hint to the profane wand somewhere in the stone age quest page.

The profane wand secret aspect, yeah I can instead make it just the trigger if it's that problematic.

**Comment by PlayfulPiano** [comment]
Back home, and I made the adjustments requested.

- Removed the "A Dangerous Trade" secret quest
- Added "50 vis" quest completion reward to the "Profane Wand" trigger quest
- Readded the Thaumometer prerequisite to Scribing Tools
- Added "Early Scribing Tool" trigger quest, requires "Profane Wand" trigger quest as a prerequisite
- Removed the "Infused Stone Processing" quest, moved info into the normal Infused Stone quest
- Removed the "Getting Shard" prereq requirements for balanced shards & crystal cluster quests
- Some quest repositioning
- Some quest description changes

**Comment by Alastors** [comment]
I disagree with chochom on the secret quest stuff

**Comment by Alastors** [comment]
Honestly, overall from face value these seem like good changes, wandskip and all

**Comment by PlayfulPiano** [comment]
> I disagree with chochom on the secret quest stuff

/shrug

I can revert it back to include the secret quest but yeah uh we should have an agreement on that lol

**Review by chochem — CHANGES_REQUESTED** [review]
much better now, one big thing left though.

the warp warnings really need to back in main progression and main quest. its just that important.

**Comment by PlayfulPiano** [comment]
Alright, PR updated hopefully for the last time now.

- Added back warp warnings to the main quest prereq path
- Separated the "Useful Plant Life" quest into multiple quests with more detailed information
- Swapped "Primal Shrooms" and "Enchanted Earth" for the plant quest related prereq
- Some quest position changes
- Added mention about the new 2.7 warp sounds

**Review by chochem — APPROVED** [review]
awesome, thanks for the patient follow-ups!

**Pull request merged**